src/components/pastAnalysis/seriesTable.tsx

```diff
@@ -26,45 +28,66 @@ export const isLargestNumber = (num: number | null | undefined, numArray: Map<st
   }
 }
 
-const SeriesTable: React.FC<SeriesTableProps> = ({ studyInstance, isProcessing }) => {
+const SeriesTable: React.FC<SeriesTableProps> = ({ studyInstance, classifications, isProcessing }) => {
   const history = useHistory();
   const { dispatch } = useContext(AppContext);
   const { series: analysisList } = studyInstance;
+
   let titles = [
     { title: (<span><br />Preview<span className='classificationText'>&nbsp;</span></span>) },
     { title: (<span><br />Image<span className='classificationText'>&nbsp;</span></span>) }
   ];
 
-  analysisList[0]?.classifications.forEach((value: number, key: string) => {
-    titles.push({ title: (<span><br /><span className='classificationText'>{key}</span></span>) }); // Adding the column titles for each analysis
-  });
+  if (classifications.length) {
+    classifications.forEach((value: string) => {
+      titles.push({ title: (<span><br /><span className='classificationText'>{value}</span></span>) }); // Adding the column titles for each analysis
```

Comment:
Replace single quotes with double quotes.
```suggestion
      titles.push({ title: (<span><br /><span className="classificationText">{value}</span></span>) }); // Adding the column titles for each analysis
```